tilelog/country.py

```diff
@@ -24,4 +24,8 @@ def country_logs(curs, date, dest):
                   "min_requests": tilelog.constants.MIN_TILE_REQUESTS})
     csvwriter = csv.writer(dest, dialect=csv.unix_dialect,
                            quoting=csv.QUOTE_NONNUMERIC)
+    # Write the header row
+    csvwriter.writerow(["country", "ips", "tps", "tps_miss"])
```

Comment:
Is there any reason [writeheader](https://docs.python.org/3/library/csv.html#csv.DictWriter.writeheader) won't work?